mimecast/plugin.spec.yaml

```diff
@@ -701,8 +701,8 @@ actions:
         - malicious
         - all
       url_to_filter:
-        title: Filter
-        description: Reguler exprestion matching to filter on. e.g. `examp` will return only URLs with the letters exap in them
+        title: Regular expression filter
+        description: Regular expression matching to filter on. e.g. `examp` will return only URLs with the letters exap in them
```

Comment:
Use the Github suggestions. I had a fix in there for `examp` vs. `exap` which is still lingering here.